### ISSUE TITLE
[BUG] Bot want to keep sleeping after sleeping

### ISSUE THREAD

**Issue by davidspeijer** [PokemonGoF/PokemonGo-Bot#5242]
### Expected Behavior

After sleep the bot continue to work

### Actual Behavior

After the bot has been sleeping, it wakes up and decide to go back to sleep. 
?? https://github.com/PokemonGoF/PokemonGo-Bot/pull/5180
?? https://github.com/PokemonGoF/PokemonGo-Bot/pull/5176

### Your FULL config.json (remove your username, password, gmapkey and any other private info)

http://pastebin.com/7e8zmb7A

### Output when issue occurred

[2016-09-06 14:54:22] [SleepSchedule] Sleeping for 00:06:17, wake at 2016-09-06 15:00:40.698931
[2016-09-06 15:00:41] [PokemonGoBot] Login procedure started.
[2016-09-06 15:00:42] [PokemonGoBot] Login successful.
[2016-09-06 15:00:42] [IncubateEggs] Eggs incubating: [3.01/5.0 km, 5.11/10.0 km] (Eggs left: 7, Incubating: 2)
[2016-09-06 15:00:44] [SleepSchedule] Sleeping for 00:08:59, wake at 2016-09-06 15:09:44.136603
[2016-09-06 15:09:44] [PokemonGoBot] Login procedure started.
[2016-09-06 15:09:46] [PokemonGoBot] Login successful.
[2016-09-06 15:09:46] [IncubateEggs] Eggs incubating: [3.01/5.0 km, 5.11/10.0 km] (Eggs left: 7, Incubating: 2)
[2016-09-06 15:09:47] [MoveToFort] Moving towards pokestop Thomas More - 0.11km
[2016-09-06 15:09:49] [SleepSchedule] Sleeping for 00:03:34, wake at 2016-09-06 15:13:24.431300

### Steps to Reproduce

Start the bot

### Other Information
OS: CentOS
 
Branch: dev
 
Git Commit: f543e3d66e502f89693b6675f494a421deb28000
 
Python Version: Python 2.7.5
 
Any other relevant files/configs (eg: path files)

**Comment by Baconnaise** [comment]
I'd love to have a flag true or false to shut this whole aspect down completely. I do things manually and when I start the bot I want it to start.

**Comment by sohje** [comment]
```
[2016-09-06 14:54:22] start
[2016-09-06 15:00:44] start
[2016-09-06 15:09:49] start
```
```js
      {
         "time": "15:00",
         "duration": "00:15",
         "time_random_offset": "00:10",
         "duration_random_offset": "00:10",
         "wake_up_at_location": ""
      }
```

**Comment by cmezh** [comment]
It happens because #5176 rewrited some of changes of #5180 where it was fixed.
My new PR is almost ready and will fix it again.

**Comment by cmezh** [comment]
Fixed with #5278
